pkg/helmcertifier/checks/checks_test.go

```diff
@@ -30,34 +31,28 @@ func TestIsHelmV3(t *testing.T) {
 	}
 
 	positiveTestCases := []testCase{
-		{description: "valid tarball, absolute path", uri: "/tmp/chart-v3.tgz"},
-		{description: "valid tarball, relative path", uri: "./chart-v3.tgz"},
-		{description: "valid tarball, http", uri: "http://www.example.com/chart-v3.tgz"},
-		{description: "valid directory, absolute path", uri: "/tmp/chart-v3"},
-		{description: "valid directory, relative path", uri: "./chart-v3"},
+		{description: "valid tarball, absolute path", uri: "testchart-0.1.0.tgz"},
 	}
 
 	for _, tc := range positiveTestCases {
 		t.Run(tc.description, func(t *testing.T) {
 			r, err := IsHelmV3(tc.uri)
 			require.NoError(t, err)
 			require.NotNil(t, r)
+			require.Equal(t, true, r.Ok)
 		})
 	}
 
 	negativeTestCases := []testCase{
-		{description: "invalid tarball, absolute path", uri: "/tmp/chart-v2.tgz"},
-		{description: "invalid tarball, relative path", uri: "./chart-v2.tgz"},
-		{description: "invalid tarball, http", uri: "http://www.example.com/chart-v2.tgz"},
-		{description: "invalid directory, absolute path", uri: "/tmp/chart-v2"},
-		{description: "invalid directory, relative path", uri: "./chart-v2"},
+		{description: "invalid tarball, absolute path", uri: "helm2testchart-0.1.0.tgz"},
```

Comment:
Please normalize test artifacts name: the valid tarball is called `testchart-0.1.0.tgz` and the invalid is called `helm2testchart-0.1.0.tgz`. I would abuse semantic versioning for this: `chart-0.1.0-v3.valid.tgz`, `chart-0.1.0-v3.invalid.no-readme.tgz`, etc.

Reply:
done.